net-misc/dahdi-tools/dahdi-tools-3.1.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 autotools
+
+DESCRIPTION="Userspace tools to configure the kernel modules from net-misc/dahdi"
+HOMEPAGE="http://www.asterisk.org"
+SRC_URI="http://downloads.asterisk.org/pub/telephony/${PN}/releases/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="ppp"
+PATCHES=( "${FILESDIR}/dahdi-nondigium-blacklist.patch" )
+
+DEPEND="dev-libs/newt
+	net-misc/dahdi
+	sys-kernel/linux-headers
+	virtual/libusb:0
+	ppp? ( net-dialup/ppp )"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf $(use_with ppp)
+}
+
+src_install() {
+	local bashcompdir="$(get_bashcompdir)"
+	local bashcmd bashcmdtarget
+
+	emake DESTDIR="${D}" bashcompdir="${bashcompdir}" udevrulesdir=/lib/udev/rules.d install
+	emake DESTDIR="${D}" install-config
+
+	dosbin patgen pattest patlooptest hdlcstress hdlctest hdlcgen hdlcverify timertest
+
+	# install init scripts
+	newinitd "${FILESDIR}"/dahdi.init2 dahdi
+	newinitd "${FILESDIR}"/dahdi-autoconf.init2 dahdi-autoconf
+	newconfd "${FILESDIR}"/dahdi-autoconf.conf2 dahdi-autoconf
+
+	# Fix up bash completion ... to Gentoo standards...
+	for bashcmd in $(sed -nre 's/^complete -F .* //p' "${ED}${bashcompdir}/dahdi"); do
```

Comment:
Is this really needed? Can't it be dealth with the eclass you inherit? 
`$(sed -nre 's/^complete -F .* //p' "${ED}${bashcompdir}/dahdi" || die)`

Reply:
make install installs the bash completion file.

From there I can either rename or add aliases.  Looking at the eclass code the update is probably the best I can do.  Happy with that?